src/planning.py

```diff
@@ -149,8 +158,9 @@ def _skeleton_generator(task: Task,
                       atoms_sequence=[init_atoms], parent=None)
     rng_prio = np.random.default_rng(seed)
     heuristic = utils.create_heuristic(CFG.task_planning_heuristic,
-                                       init_atoms, task.goal, ground_nsrts)
-    hq.heappush(queue, (heuristic(root_node.pyperplan_facts),
+                                       init_atoms, task.goal, ground_nsrts,
```

Comment:
Lots of effort in this file to get the predicates to this point. But why does the pyperplan heuristic need to be told the predicates? Can't it extract them from the NSRTs via `extract_preds_and_types`?

Reply:
resolved on slack